### ISSUE TITLE
BestBuy Canada Support

### ISSUE THREAD

**Issue by AdrisAz** [zeldridge/best-buy-sniper#20]
By any chance would you be able to update this to support the Canadian version of BestBuy?

**Comment by OlivierMartinez** [comment]
@4cylinder except the method:  hasItemBeenAddedToCart()wont work hasItemBeenAddedToCart() cause /api/v1/addToCart doesn't exist in the Canadian one.

**Comment by 4cylinder** [comment]
@OlivierMartinez Press F12 in Chrome, open the Network tab, and go through the process of adding any random product to the cart. You'll find the API link easily enough.

I'll give you a hint: It contains the word "basket".

Also, you don't have to use the API to verify if an item is in the cart. You can navigate to the actual cart page and inspect the CSS selectors there.

**Comment by OlivierMartinez** [comment]
@OlivierMartinez yeah i saw (/api/basket/v2/baskets) but the POST works ? Pretty sure the hasItemBeenAddedToCart() method won't work as /api/basket/v2/baskets seems to return an error everytime. Anyway this bot seems to have been abandoned by its creators. Some guy has created a fork i'll give a look !
